src/backend/api/backend.did

```diff
@@ -247,6 +247,56 @@ type UpdateProposalReviewResponse = variant {
   err : Err;
 };
 
+type ReviewCommitState = variant {
+  reviewed : record {
+    matches_description : bool;
+    comment : opt text;
+    highlights : vec text;
```

Comment:
"highlights" should be optional, because this is what will be added to the review summary if the reviewer adds it.

Is "comment" the same as "summary" from the wireframes? This is the one that we always want.

Reply:
> "highlights" should be optional, because this is what will be added to the review summary if the reviewer adds it.

Since highlights is a vector, what's the point of having it optional if it can just be empty? In addition, candid js bindings for `opt` and `vec` are both generated as an array.

> Is "comment" the same as "summary" from the wireframes? This is the one that we always want.

Looking at the wireframes, it looks like users can just set the `matches_description` flag on the commit, without having to necessarily add a comment.
This is effectively what happens already with the current reviews, when we just say "Matches description" or "Same as description" in the commit review comment.

Reply:
We discussed to just make the `matches_description` a `Option<bool>`